app/main.py

```diff
@@ -1,2 +1,46 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, length: int = 0, capacity: int = 8) -> None:
+        self.length = length
+        self.capacity = capacity
+        self.load_factor = int(self.capacity * 2 / 3)
+        self.hash_table: list = [None] * self.capacity
+
+    def resize(self) -> None:
+        self.length = 0
+        self.capacity *= 2
+        self.load_factor = int(self.capacity * (2 / 3))
+        prev_hash_table = self.hash_table
+        self.hash_table: list = [None] * self.capacity
+        for item in prev_hash_table:
+            if item:
+                key, _, value = item
+                self.__setitem__(key, value)
+
+    def __setitem__(self, key: Any, value: Any) -> None:
+        if self.length > self.load_factor:
```

Comment:
And what if length is equal to load_factor and you write new element? I believe it’s better to check equality here
```suggestion
        if self.length == self.load_factor:
```

Reply:
pls resolve comment